lib/features/copy-paste/BpmnCopyPaste.js

```diff
@@ -122,6 +141,9 @@ export default function BpmnCopyPaste(
       }
     }
 
+    // resolve references references e.g. default sequence flow
```

Comment:
can one `referenced` word be removed (typo)?

Reply:
Which one though? The first or the second one? :cake: